client/a8c-for-agencies/sections/marketplace/lib/hosting.tsx

```diff
@@ -0,0 +1,149 @@
+import {
+	A4A_MARKETPLACE_HOSTING_LINK,
+	A4A_MARKETPLACE_HOSTING_PRESSABLE_LINK,
+	A4A_MARKETPLACE_HOSTING_WPCOM_LINK,
+} from 'calypso/a8c-for-agencies/components/sidebar-menu/lib/constants';
+import { APIProductFamilyProduct } from 'calypso/state/partner-portal/types';
+
+/**
+ * Get the cheapest plan from a list of plans
+ * @param {APIProductFamilyProduct[]} plans - List of plans
+ * @returns {APIProductFamilyProduct} - Cheapest plan
+ */
+export function getCheapestPlan( plans: APIProductFamilyProduct[] ) {
+	return plans.reduce( ( cheapestPlan: APIProductFamilyProduct, plan: APIProductFamilyProduct ) => {
+		if ( Number( plan.amount ) < Number( cheapestPlan.amount ) ) {
+			return plan;
+		}
+		return cheapestPlan;
+	}, plans[ 0 ] );
+}
+
+/**
+ * Get the URL for a hosting provider
+ * @param {string} slug - Hosting provider slug
+ * @returns {string} - Hosting provider URL
+ */
+export function getHostingPageUrl( slug: string ) {
+	switch ( slug ) {
+		case 'pressable-hosting':
+			return A4A_MARKETPLACE_HOSTING_PRESSABLE_LINK;
+		case 'wpcom-hosting':
+			return A4A_MARKETPLACE_HOSTING_WPCOM_LINK;
+	}
+
+	return A4A_MARKETPLACE_HOSTING_LINK;
+}
+
+/**
+ * Get the logo for a hosting provider
+ * @param {string} slug - Hosting provider slug
+ * @returns {Element} - Hosting provider logo
+ */
+export function getHostingLogo( slug: string ) {
+	switch ( slug ) {
+		case 'pressable-hosting':
+			return (
+				<svg
+					width="198"
```

Comment:
I think we can keep these SVG files here: [client/assets/images](https://github.com/Automattic/wp-calypso/blob/e8d66114b31392a90f4fb08d1601d98c3b456c95/client/assets/images). What do you think?